live-examples/js-examples/array/array-reverse.html

```diff
@@ -7,8 +7,7 @@
 console.log('reversed:', reversed);
 // expected output: "reversed:" Array ["three", "two", "one"]
 
-/* Careful: reverse is destructive. It also changes
-the original array */
+/* Careful: reverse is destructive -- it changes the original array. */
```

Comment:
@wbamberg — The initial reason for this PR was to reduce the confusion in the comment above. The previous wording, “Careful: reverse is destructive. It also changes the original array”, is confusing because it sounds like the “also changes the original array” part is unrelated to the “is destructive” part. My change makes it clear that the explanation for “is destructive” is that it indeed changes the original array. Would you mind merging this in? Thanks.

Reply:
Oh sorry, I didn't notice that. I agree with your reasoning here. If you can update this PR to only include that change, I'd be very happy to merge it.